src/components/layouts/Header/index.jsx

```diff
@@ -3,10 +3,26 @@ import s from './Header.module.scss';
 import { NavLink } from 'react-router-dom';
 import { useState } from 'react';
 import { useIntl } from 'react-intl';
+import { Button } from 'src/components/common/Button';
+import { Modal } from 'src/components/common/Modal';
+import { useDimension } from 'src/hooks/useDimension';
 
 export const Header = () => {
   const { formatMessage } = useIntl();
   const [triggerBurger, setTriggerBurger] = useState(false);
+  const [isOpenModal, setIsOpenModal] = useState(false);
+  const { width, height } = useDimension();
+  console.log(width, height);
+
+  const tabletWidth = 768;
```

Comment:
create constants for breakpoints in global constants since we will reuse them and call them properly